openmls/src/messages/mod.rs

```diff
@@ -241,6 +241,7 @@ impl Signable for GroupInfoTBS {
 /// } GroupInfo;
 /// ```
 #[derive(Debug, PartialEq, Clone)]
+#[cfg_attr(feature = "test-utils", derive(TlsDeserialize))]
```

Comment:
This is needed because `MlsMessageOutBody` derives `TlsDeserialize` under the same conditions and uses `GroupInfo`.